teams/Module.md

```diff
@@ -1,43 +1,18 @@
 ## Overview
-Ballerina connector for Microsoft Teams is connecting to MS Teams platform API in Microsoft Graph via Ballerina 
-language easily. It provides capability to perform basic functionalities provided in MS Teams such as Sending messages, 
+Ballerina connector for Microsoft Teams is connecting to MS Teams platform API in Microsoft Graph via Ballerina language. 
```

Comment:
Let's rephrase this as "Microsoft Teams connector provides the capability to connect to the MS Teams platform API in Microsoft Graph via [Ballerina](https://ballerina.io/). It allows you to perform basic functionalities provided in MS Teams such as sending messages, viewing messages, creating teams, channels, and chats, deleting and updating resources, etc., programmatically. "

Reply:
Fixed this